frontend/src/components/stats/ActiveNearAppsUsersByDate.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from "react";
+import ReactEcharts from "echarts-for-react";
+import * as echarts from "echarts";
+import { useTranslation } from "react-i18next";
+
+import { AccountsByDate } from "../../libraries/explorer-wamp/stats";
+
+import { Props } from "./TransactionsByDate";
+
+const ActiveNearAppsUsersByDate = ({ chartStyle }: Props) => {
+  const { t } = useTranslation();
+  const [activeAccountsByWeek, setWeekAccounts] = useState(Array());
+  const [week, setWeek] = useState(Array());
+
+  useEffect(() => {
+    // TODO(sparkyrider): Put a proper implementation in near-analytics after User Activity NEP lands and get implemented on indexer-for-explorer side.
+    // Meanwhile, @sparkyrider will manually submit the new values weekly in PRs to NEAR Explorer.
+    // XXX: Hardcoded values were generated by this query:
+    // select to_timestamp(receipt_included_in_block_timestamp / 1000000000)::date as date
+    //  ,count(*) as accountscount
+    //  from action_receipt_actions
+    // where receipt_receiver_account_id = 'v1.nearapps.near'
+    //   and to_timestamp(receipt_included_in_block_timestamp / 1000000000) > NOW() - interval '60 days'
+    // group by to_timestamp(receipt_included_in_block_timestamp / 1000000000)::Date;
+    let accounts = [
```

Comment:
Should it be `const`?
Let's put it out of the component to prevent creating the object on every render.